lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -690,8 +698,10 @@ WasmBytecodeGenerator::EmitCall()
         switch (info.type)
         {
         case WasmTypes::F32:
-            Assert(wasmOp != wnCALL_IMPORT);
-            // REVIEW: support FFI call with f32 params?
+            if (wasmOp == wnCALL_IMPORT) 
+            {
+                throw WasmCompilationException(_u("External calls with float argument NYI"));
```

Comment:
The main reason I put this is because there is a test in the WebAssembly/testsuite that hit this assert. Therefore, I assume it is possible to do in Wasm.
I haven't thought on how to do it yet. This is to convert an assert into a runtime error.